Packages/cc.starlessnight.unity-jsb/Source/Binding/Values_get_var.cs

```diff
@@ -134,6 +134,16 @@ public static JSValue js_push_var(JSContext ctx, object o)
                 return JSApi.JS_UNDEFINED;
             }
 
+            if (o is ScriptValue ss)
+            {
+                return JSApi.JS_DupValue(ctx, ss);
+            }
+
+            if (o is JSValue jv)
+            {
+                return JSApi.JS_DupValue(ctx, jv);
+            }

```

Comment:
Convert `ScriptValue` to `JSValue` when passing it back to Javascript. Pass `JSValue` as is.